lib/mcp/railtie.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'logger'
+require 'fileutils'
+require_relative '../mcp/server'
+
+module FastMcp
+  # Railtie for integrating Fast MCP with Rails applications
+  class Railtie < Rails::Railtie
+    # Add tools and resources directories to autoload paths
+    initializer 'fast_mcp.setup_autoload_paths' do |app|
+      app.config.autoload_paths += %W[
+        #{app.root}/app/tools
+        #{app.root}/app/resources
```

Comment:
Does it make sense to scope by `mcp` or `fast_mcp` namespace?
eg. 
```suggestion
        #{app.root}/app/fast_mcp/tools
        #{app.root}/app/fast_mcp/resources
```
The `app` folder tends to be polluted (for big projects), and `tools` and `resources` are such common concepts.

`app/lib/mcp` works too.

Reply:
For greenfield projects, autoloading of anything in `app` folder works automagically (with Zeitwerk). Even for brownfield projects, as of Rails 6, the Zeitwerk is the default autoloader.